plugin/Gui/AdminPageApiSettings.php

```diff
@@ -144,6 +145,24 @@ private function addFormModelGoogleMapsKey()
 		$this->addFormModel($pFormModel);
 	}
 
+	private function addFormModelGoogleBotSettings()
+	{
+		$labelGoogleBotIndexPdfExpose = __('Index Pdf Expose', 'onoffice-for-wp-websites');
```

Comment:
The correct term for "Pdf Expose" would be `PDF brochure`. That's something that's confusing for now but should be used more, especially across the code.

Reply:
Ok I will change "Pdf Expose" to "PDF brochure"